### PR TITLE
sys-boot/efibootmgr: EAPI 7, version bump

### PR DESCRIPTION
**Pull request by telans**
Closes: https://bugs.gentoo.org/587094
Closes: https://bugs.gentoo.org/673536
Closes: https://bugs.gentoo.org/557248
Package-Manager: Portage-3.0.13, Repoman-3.0.2
Signed-off-by: Theo Anderson <telans@posteo.de>

### CODE REVIEW

File: sys-boot/efibootmgr/efibootmgr-17.ebuild [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	sed -i '/extern int efi_set_verbose/d' src/efibootmgr.c || die
+}
+
+src_configure() {
+	tc-export CC
+	export EFIDIR="Gentoo"
+}
+
+src_compile() {
+	PKG_CONFIG="$(tc-getPKG_CONFIG)" emake
```
Comment (floppym) [inline sys-boot/efibootmgr/efibootmgr-17.ebuild:38]: It looks like you swapped the order of emake and PKG_CONFIG here. This has a negative effect on the build process.

In Make.defaults, PKG_CONFIG is set using the `=` operator. This means that environment variables will not override it, but command line arguments will.

File: sys-boot/efibootmgr/efibootmgr-17.ebuild [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-clang.patch" )
+
+src_prepare() {
+	default
+	sed -i 's/-Werror //' Make.defaults || die
+	# Remove conflict with =sys-libs/efivar-37.
+	# See: https://github.com/rhboot/efibootmgr/issues/128
+	sed -i '/extern int efi_set_verbose/d' src/efibootmgr.c || die
```
Comment (floppym) [inline sys-boot/efibootmgr/efibootmgr-17.ebuild:29]: A patch would be preferable here.

File: sys-boot/efibootmgr/efibootmgr-17.ebuild (hunk 1 above)
Comment (telans) [reply sys-boot/efibootmgr/efibootmgr-17.ebuild:38]: Sorry, I meant to put that back.

File: sys-boot/efibootmgr/efibootmgr-17.ebuild (hunk 2 above)
Comment (telans) [reply sys-boot/efibootmgr/efibootmgr-17.ebuild]: Done

**Pull request closed without merging**